src/backend/expungeservice/expunger.py

```diff
@@ -24,7 +24,7 @@ def run(self) -> bool:
         """
         open_cases = [case for case in self.record.cases if not case.closed()]
         if len(open_cases) > 0:
-            case_numbers = ",".join([case.case_number for case in open_cases])
+            case_numbers = ",".join([f"[{case.case_number}]" for case in open_cases])
```

Comment:
While we're here you can fix this space:
```suggestion
            case_numbers = ", ".join([f"[{case.case_number}]" for case in open_cases])
```